pkg/config/config.go

```diff
@@ -20,11 +20,12 @@ type imagesReference struct {
 
 // OperatorConfig contains configuration values for templating resources
 type OperatorConfig struct {
-	ManagedNamespace string
-	ControllerImage  string
-	CloudNodeImage   string
-	IsSingleReplica  bool
-	Platform         configv1.PlatformType
+	ManagedNamespace   string
+	ControllerImage    string
+	CloudNodeImage     string
+	IsSingleReplica    bool
+	InfrastructureName string
```

Comment:
Would be better to name this var `ClusterName` as it is later passed to the `--cluster-name` flag.

Reply:
Would prefer keep `InfrastructureName` consistent across our code since it titled this way in infrastructure resource. Renamed variable in manifests for better consistency.